catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/RemoveCommandTest.java

```diff
@@ -128,4 +171,21 @@ protected SolrCacheMBean getCacheProxy() {
 
         return metacards;
     }
+
+    private java.util.List<Result> getResultList(int amount) {
+        java.util.List<Result> results = new ArrayList<>();
```

Comment:
Should be able to remove `java.util.` section of List type. Might need to put cursor on "List" in IntelliJ and then press `ALT+ENTER` to add import.